src/components/environment-variables/use-save-env-var.js

```diff
@@ -0,0 +1,28 @@
+import { usePatchJson } from '../../effects';
+import PropTypes from 'prop-types';
+import updatableEnvVar from '../../models/environment-variable/updatable-environment-variable';
+
+const useSaveEnvVar = ({
+  appName,
+  envName,
+  componentName,
+  updatableEnvVars,
+}) => {
+  const encAppName = encodeURIComponent(appName);
+  const encEnvName = encodeURIComponent(envName);
+  const encComponentName = encodeURIComponent(componentName);
+  const path = `/applications/${encAppName}/environments/${encEnvName}/components/${encComponentName}/envvars`;
+
+  return usePatchJson(path, (props) => {
+    return props.updatableEnvVars;
```

Comment:
ref above comment: props.updatableEnvVars refers to the object you pass as argument to saveFunc in environment-variable-list.js. Since the request body of path is onløy the list of variables, you only need to pass the updatableEnvVars value in saveFunc and return it in the function in usePatchJson

Reply:
Fixed